EventFilter/TotemRawToDigi/test/analyze_vfat_frames.py

```diff
@@ -0,0 +1,43 @@
+import FWCore.ParameterSet.Config as cms
+import FWCore.ParameterSet.VarParsing as VarParsing
+
+process = cms.Process("AnalyzeVFATFrames")
+
+# default options
+options = VarParsing.VarParsing ('analysis')
+options.inputFiles= 'file:/afs/cern.ch/user/j/jkaspar/public/run268608_ls0001_streamA_StorageManager.root',
```

Comment:
Does it mean that nobody in CTPPS community has /store/group on CERN eos?
The file fragment that you are using is supposedly in
/store/data/Commissioning2016/MiniDaq/RAW/v1/000/268/608/00000/56702640-03FC-E511-96DC-02163E01391D.root

I was able to get to it with xrootd:
``` 
edmFileUtil -l -e  root://cms-xrd-global.cern.ch/\
/store/data/Commissioning2016/MiniDaq/RAW/v1/000/268/608/00000/56702640-03FC-E511-96DC-02163E01391D.root
```